File: data/src/main/java/com/github/gmetal/data/entity/MovieDatesEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class MovieDatesEntity {

    @JsonField(name = "minimum")
    public String mMinimum;

    @JsonField(name = "maximum")
    public String mMaximum;
}

File: data/src/main/java/com/github/gmetal/data/entity/MovieDatesEntity.kt
package com.github.gmetal.data.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class MovieDatesEntity {

    @SerialName("minimum")
    var mMinimum: String? = null

    @SerialName("maximum")
    var mMaximum: String? = null
}
